subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/TransformInfo.java

```diff
@@ -200,11 +150,16 @@ protected ResolvedTransformInputs resolveInputs() {
                 } else {
                     failure = new DefaultLenientConfiguration.ArtifactResolveException("artifacts", artifactTransformer.getDisplayName(), "artifact transform", failures);
                 }
-                return new ResolvedTransformInputs(failure);
-            } else {
-                ResolvedArtifactResult artifact = Iterables.getOnlyElement(visitor.getArtifacts());
-                return new ResolvedTransformInputs(ImmutableList.of(artifact.getFile()));
+                this.failure = failure;
+                this.result = Collections.emptyList();
+                return;
             }
+            ResolvedArtifactResult artifact = Iterables.getOnlyElement(visitor.getArtifacts());
+
+            TransformArtifactOperation operation = new TransformArtifactOperation(artifact.getId(), artifact.getFile(), artifactTransformer, BuildOperationCategory.TASK);
```

Comment:
What happens if we use `UNCATEGORIZED` here?

Reply:
It disappears from the verbose console.